x/programs/cmd/simulator/vm/actions/program_execute.go

```diff
@@ -86,7 +86,6 @@ func (t *ProgramExecute) Execute(
 	// TODO: get cfg from genesis
 	cfg, err := runtime.NewConfigBuilder().
 		WithDebugMode(true).
-		WithBulkMemory(true).
 		// only required for Wasi support exposed by testing only.
```

Comment:
nit: my fault but could we remove this dangling doc line as it is no longer useful. :pray: